[3.3.0RC1] KeyStoreException in DeviceCredentialUtils.createKey

GPlay console
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performResumeActivity (ActivityThread.java:3790)
  at android.app.ActivityThread.handleResumeActivity (ActivityThread.java:3830)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3038)
  at android.app.ActivityThread.-wrap11 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1696)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6944)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:327)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1374)
Caused by: java.security.ProviderException: 
  at android.security.keystore.AndroidKeyStoreKeyGeneratorSpi.engineGenerateKey (AndroidKeyStoreKeyGeneratorSpi.java:365)
  at javax.crypto.KeyGenerator.generateKey (KeyGenerator.java:604)
  at com.owncloud.android.utils.DeviceCredentialUtils.createKey (DeviceCredentialUtils.java:97)
  at com.owncloud.android.ui.activity.RequestCredentialsActivity.onResume (RequestCredentialsActivity.java:69)
  at android.app.Instrumentation.callActivityOnResume (Instrumentation.java:1361)
  at android.app.Activity.performResume (Activity.java:7352)
  at android.app.ActivityThread.performResumeActivity (ActivityThread.java:3765)
Caused by: android.security.KeyStoreException: 
  at android.security.KeyStore.getKeyStoreException (KeyStore.java:1112)
  at android.security.keystore.AndroidKeyStoreKeyGeneratorSpi.engineGenerateKey (AndroidKeyStoreKeyGeneratorSpi.java:366)
```

--- a/src/main/java/com/owncloud/android/utils/DeviceCredentialUtils.java
+++ b/src/main/java/com/owncloud/android/utils/DeviceCredentialUtils.java
@@ -39,6 +39,7 @@ import java.security.KeyStore;
 import java.security.KeyStoreException;
 import java.security.NoSuchAlgorithmException;
 import java.security.NoSuchProviderException;
+import java.security.ProviderException;
 import java.security.UnrecoverableKeyException;
 import java.security.cert.CertificateException;
 
@@ -101,7 +102,7 @@ public final class DeviceCredentialUtils {
             keyGenerator.generateKey();
         } catch (NoSuchAlgorithmException | NoSuchProviderException
                 | InvalidAlgorithmParameterException | KeyStoreException
-                | CertificateException | IOException e) {
+                | CertificateException | ProviderException | IOException e) {
             Log_OC.e(TAG, "Exception: " + e.getMessage());
         }
     }

